pkg/controller/handlers/handlers.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/metadata/labels"
 )
 
-//nolint:ireturn
+
```

Comment:
this seems to have been removed by mistake

Reply:
yeah, it is back and forth, but i will fix it along with e2e tests